### PR TITLE
Fixed all errors inside combat_tactics

### PR DESCRIPTION
**Pull request by Ramit110**
The only issues seen in vanilla ``00_combat_tactics.txt`` should now be localisation errors.

### CODE REVIEW

File: links.cwt [hunk 1]
```diff
@@ -178,7 +178,7 @@ links = {
     enemy = {
         desc = "Scopes to the enemy in a siege"
         input_scopes = siege
-        output_scope = character
+        output_scope = flank
```
Comment (UristMcDorf) [inline links.cwt]: Change that to "siege", there's no flank scope defined for CKII in the extension's code.

Comment (Ramit110) [reply links.cwt]: Done.

File: triggers.cwt [hunk 2]
```diff
@@ -3483,4 +3484,47 @@ alias[trigger:hidden_tooltip] = {
...
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:<special_troop_type>] = float
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:days] = int
```
Comment (tboby) [inline triggers.cwt:3507]: Do we think these are triggers that work everywhere (in siege scope)?
They're not combat_tactics specific somehow right?

Comment (Ramit110) [reply triggers.cwt:3507]: I don't particularly know, I haven't seen siege scope being used much outside of combat_tactics

Comment (Ramit110) [reply triggers.cwt:3507]: Should I put this in issues?

Comment (tboby) [reply triggers.cwt:3507]: Nah, I've had a look at vanilla and they look like real triggers, so it's fine :)

**Pull request merged**